### ISSUE TITLE
[Frontiers] How to get modded save file to replace save file before mods.

### ISSUE THREAD

**Issue by Nitroboltt** [thesupersonic16/HedgeModManager#545]
So I want to have my 100% save data and arcade data be on the steam cloud but I had save file redirect on the whole time. I know in generations you could just take "cpkredir.sav", rename it and have it replace the "sonic.sav" but it's very different in frontiers. The modded save file this time seems to be in "cpkredir.ini" which I can't do anything with. I'm very inexperienced so does anyone know how I could access the modded save file and have it replace the non-modded one?

**Comment by Nitroboltt** [comment]
Nevermind, figured out the original location of the save file.

**Comment by AUniverseT** [comment]
Im having this issue and I dont understand how you fixed it

**Comment by Nitroboltt** [comment]
> Im having this issue and I dont understand how you fixed it
Well, the first thing I did was to ensure that "Enable save file redirection" was turned off in HedgeModManager. I figured out where the actual save file for Frontiers is, "%APPDATA%\Sega\SonicFrontiers\steam\<user-id>". I replaced the save file with the modded save file in the steam folder for frontiers. Which was in D:\SteamLibrary\steamapps\common\SonicFrontiers for me. Basically, the modded save file will be wherever your game folder is. Just cut that folder and past it into the original location of the save file.
